### ISSUE TITLE
Issue with the use of yaml.load

### ISSUE THREAD

**Issue by marioa** [softwaresaved/carpentry-data-analysis#35]
In:
```
amy_data_extract.py:373: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  amy_credentials_yaml = yaml.load(stream)
```
See:

* https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation

**Comment by anenadic** [comment]
See the following on how to solve:
https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation

**Comment by anenadic** [comment]
Closed via c3a6cb23fbcafd16d5c57abad215705bea7771a6
